plone/app/content/browser/contents/__init__.py

```diff
@@ -146,26 +146,43 @@ def get_actions(self):
     def get_columns(self):
         # Base set of columns
         columns = {
-            'id': translate(_('ID'), context=self.request),
-            'ModificationDate': translate(_('Last modified'), context=self.request),  # noqa
-            'EffectiveDate': translate(_('Publication date'), context=self.request),  # noqa
-            'ExpirationDate': translate(_('Expiration date'), context=self.request),  #noqa
             'CreationDate': translate(_('Created on'), context=self.request),  # noqa
-            'review_state': translate(_('Review state'), context=self.request),  # noqa
-            'Subject': translate(_('Tags'), context=self.request),
-            'portal_type': translate(_('Type'), context=self.request),
-            'is_folderish': translate(_('Folder'), context=self.request),
+            'Creator': translate(_('Creator'), context=self.request),
```

Comment:
This was marked as ``_unsage_metadata`` in plone.app.content. I don't think that information is necessarily unsafe...